.circleci/config.yml

```
@@ -192,6 +192,102 @@ jobs:
               -timeout 60s \
                 ~/.local/bin/pytest
 
+  # ---- Tray jobs (k8s) ----
+  tray:
+    machine:
+      image: ubuntu-2004:202201-02
+      # Prevent cache-related issues
+      docker_layer_caching: false
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - *generate-version-file
+      - run:
+          name: Build production image
+          command: docker build -t joanie:${CIRCLE_SHA1} --target production .
+      - run:
+          name: Check built image availability
+          command: docker images "joanie:${CIRCLE_SHA1}*"
+      # Login to DockerHub to Publish new images
+      #
+      # Nota bene: you'll need to define the following secrets environment vars
+      # in CircleCI interface:
+      #
+      #   - DOCKER_USER
+      #   - DOCKER_PASS
+      - run:
+          name: Login to DockerHub
+          command: echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
```

:warning: the `DOCKER_USER` environment variable is used by the Arnold CLI/Docker image! Overriding it will break Arnold's execution as the user won't exist in the running container:

```suggestion
          command: echo "$DOCKER_HUB_PASS" | docker login -u "$DOCKER_HUB_USER" --password-stdin
```